src/templates/settings.twig

```diff
@@ -28,6 +28,22 @@
     suggestEnvVars: true
 }) }}
 
+{% if cdn_determined_from_env %}
+    {% if cdn_enabled_from_env %}
+        <input type="checkbox" name="enableCDNCacheInvalidation" value="1" checked disabled>
```

Comment:
No need to have the `if cdn_enabled_from_env` check, just put the checkbox and add a conditional directly on the attribute itself: `{{ cdn_enabled_from_env ? 'checked' }}`